httpclient/httpclient.go

```diff
@@ -41,9 +53,14 @@ type Client struct {
 	acceptType            string
 }
 
+// New creates a client configured with the config param
 func New(cfg Config) *Client {
 	t := http.DefaultTransport.(*http.Transport).Clone()
+	if cfg.MaxConnectionsPerHost == 0 {
+		cfg.MaxConnectionsPerHost = 10
+	}
 	t.MaxConnsPerHost = cfg.MaxConnectionsPerHost
+	t.MaxIdleConnsPerHost = cfg.MaxConnectionsPerHost
```

Comment:
Should this always use the same value as above?

Reply:
I tried to think of a completing reason for them to be different, and failed